lib/keen/styles/variables.scss

```diff
@@ -33,41 +33,33 @@ $ui-input-height: rem(32px) !default;
 $ui-input-margin-bottom: rem(16px) !default;
 
 // Input label
-$ui-input-label-color: $secondary-text-color !default;
+
 $ui-input-label-color--hover: black !default;
-$ui-input-label-color--active: $brand-primary-color !default;
 $ui-input-label-color--invalid: $md-red-800 !default;
 $ui-input-label-font-size: rem(15px) !default;
 $ui-input-label-line-height: normal !default;
 $ui-input-label-margin-bottom: 0 !default;
 $ui-input-label-top--inline: rem(23px) !default;
 
 // Input text
-$ui-input-text-color: $primary-text-color !default;
 $ui-input-text-font-size: rem(16px) !default;
 $ui-input-text-color--disabled: $disabled-text-color !default;
-$ui-input-text-color--invalid: $md-red !default;
 
 // Input border
-$ui-input-border-color: $secondary-text-color !default;
 $ui-input-border-color--hover: black !default;
-$ui-input-border-color--active: $brand-primary-color !default;
 $ui-input-border-color--invalid: $md-red !default;
 $ui-input-border-width: 1px !default;
 $ui-input-border-width--active: 1.5px !default;
 $ui-input-border-style--disabled: dotted !default;
 
 // Input icons
 $ui-input-icon-color: $secondary-text-color !default;
-$ui-input-icon-color--active: $brand-primary-color !default;
 $ui-input-icon-opacity--disabled: 0.6 !default;
 $ui-input-icon-margin-right: rem(12px) !default;
 $ui-input-icon-margin-top: rem(4px) !default;
 $ui-input-icon-margin-top--with-label: rem(24px) !default;
 
 // Input buttons (dropdown button icon, clear input button icon)
-$ui-input-button-color: $secondary-text-color !default;
-$ui-input-button-color--hover: $primary-text-color !default;
```

Comment:
This is another usage where `$primary-text-color` is fine to leave as is, so `$ui-input-button-color--hover` can be put back.